backend/src/LocalExec/StdLib.fs

```diff
@@ -17,6 +17,7 @@ let typeRenames : StdLib.TypeRenames =
 let contents : StdLib.Contents =
   StdLib.combine
     [ Libs.Packages.contents
+      Libs.Packages2.contents
```

Comment:
follow-up: probably worth renaming Packages2 to something else

Reply:
Good idea, any thoughts?